framework/resources/Datasources/ctas/create_pruning.ddl

```diff
@@ -0,0 +1,33 @@
+create table `/DRILL_6173/tab1/1` (itm, y, m, category)  partition by (y, m, category) as  select cast(columns[1] as int) itm, cast(columns[2] as int) y, cast(columns[3] as int) m, cast(columns[4] as int) category from `PRTTN.csv` where columns[0]='tab1' and columns[2]=1987 and columns[3]=5  and columns[4]=1 ; 
```

Comment:
@denysord88 , can you confirm if these will be written into the CTAS workspace and not to the root dir on Hadoop?

Reply:
Schema is set in the json file. Also I removed first "/" from table names.
Tables will be written into this path /drill/testdata/ctas/parquet/DRILL_6173/